### PR TITLE
docs: Move sandbox docker install/git clone instructions to an include

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message: docs: Move sandbox docker install/git clone instructions to an include
Additional Description:

This will make it easier to ensure that the sandboxes are consistent, reduce the code a little, and make it easier to update

also does a bit of whitespace cleanup (as my editor is configured to do that)

also switches ordering ssh/https of git clone instructions (re #13213)

Risk Level:
Testing:
Docs Changes:
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:] touch #13213

### CODE REVIEW

File: docs/root/start/sandboxes/cache.rst [hunk 1]
```diff
@@ -28,21 +28,7 @@ Running the Sandbox
 The following documentation runs through the setup of an Envoy cluster organized
 as is described in the image above.
```
Comment (phlax) [inline docs/root/start/sandboxes/cache.rst:29]: remove this line

File: docs/root/start/sandboxes/_include/docker-install.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
+**Step 1: Install Docker**
```
Comment (phlax) [inline docs/root/start/sandboxes/_include/docker-install.rst:1]: add an intro line - similar to one of the ones removed below

File: docs/root/start/sandboxes/cors.rst [hunk 3]
```diff
@@ -31,21 +31,7 @@ Running the Sandboxes
 
 The following documentation runs through the setup of both services.
```
Comment (phlax) [inline docs/root/start/sandboxes/cors.rst:32]: remove this line

File: docs/root/start/sandboxes/csrf.rst [hunk 4]
```diff
@@ -32,21 +32,7 @@ Running the Sandboxes
 
 The following documentation runs through the setup of both services.
```
Comment (phlax) [inline docs/root/start/sandboxes/csrf.rst:33]: remove

File: docs/root/start/sandboxes/front_proxy.rst [hunk 5]
```diff
@@ -32,21 +32,7 @@ Running the Sandbox
 The following documentation runs through the setup of an Envoy cluster organized
```
Comment (phlax) [inline docs/root/start/sandboxes/front_proxy.rst:32]: remove

**Comment by mattklein123** [comment]
Needs a main merge.

/wait

File: docs/root/start/sandboxes/_include/docker-env-setup.rst [hunk 6]
```diff
@@ -0,0 +1,21 @@
+The following documentation runs through the setup of Envoy described above
```
Comment (phlax) [inline docs/root/start/sandboxes/_include/docker-env-setup.rst:1]: fullstop

**Review by mattklein123 — APPROVED** [review]
Awesome cleanup, thanks.

**Pull request merged**